Formula/ragel.rb

```diff
@@ -8,7 +8,7 @@ class Ragel < Formula
   # TODO: Return to using `url :homepage` once the SSL certificate verification
   # issue is resolved on the upstream server.
   livecheck do
-    url "https://www.colm.net/open-source/ragel/"
+    url :homepage
```

Comment:
Thank for this! Maybe a good time to delete the comment above at the same time?

Reply:
Done. 👍🏼